lua/lint/linters/sqruff.lua

```diff
@@ -0,0 +1,43 @@
+local severities = {
+  Error = vim.diagnostic.severity.ERROR,
+  Warning = vim.diagnostic.severity.WARN,
+}
+
+return {
+  cmd = "sqruff",
+  args = {
+    "lint",
+    "--format=json",
+    "-",
+  },
+  ignore_exitcode = true,
+  parser = function(output, _)
+    if vim.trim(output) == "" or output == nil then
+      return {}
+    end
+
+    if not vim.startswith(output, "{") then
+      vim.notify(output)
+      return {}
+    end
+
+    local decoded = vim.json.decode(output)
+    local diagnostics = {}
+    local messages = decoded["<string>"]
+
+    for _, msg in ipairs(messages or {}) do
+      table.insert(diagnostics, {
+        lnum = msg.range.start.line - 1,
+        end_lnum = msg.range["end"].line - 1,
+        col = msg.range.start.character - 1,
+        end_col = msg.range["end"].character - 1,
+        message = msg.message,
+        code = msg.source, -- TODO: https://github.com/quarylabs/sqruff/issues/1219
```

Comment:
```suggestion
        -- code not provided: https://github.com/quarylabs/sqruff/issues/1219
```

Reply:
I will follow up with a new PR, if the issue is resolved upstream